packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -560,6 +562,44 @@ describeWithDOM('mount', () => {
     });
   });
 
+  describeIf(is('>= 16.6'), 'Suspense & lazy', () => {
+    it('finds Suspense and its children when no lazy component', () => {
+      class Component extends React.Component {
+        render() {
+          return (
+            <div>test</div>
+          );
+        }
+      }
+      const SuspenseComponent = () => (
+        <Suspense fallback={false}>
+          <Component />
+        </Suspense>
+      );
+
+      const wrapper = mount(<SuspenseComponent />);
+
+      expect(wrapper.find('SuspenseComponent')).to.have.lengthOf(1);
```

Comment:
`wrapper` here *is* the SuspenseComponent, so `find` isn't the proper way to assert on it.
```suggestion
      expect(wrapper.is(SuspenseComponent)).to.equal(true);
```